### PR TITLE
PAAS-1962: One-shot package to apply 8.0.3.2 patch

### PR DESCRIPTION
**Pull request by kevinvandel**
JIRA issue: https://jira.jahia.org/browse/PAAS-1962

### CODE REVIEW

File: packages/one-shot/paas-1962-apply-8032-patch.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  - installPatch
+  - checkPatchInstallation
+
+actions:
+  checkJahiaVersion:
+    - getJahiaVersion
+    - if (${globals.jahiaVersion} != "8.0.3.0"):
+        - return:
+            type: success
+            message: "No unomi linked"
```
Comment (laurentfufu) [inline packages/one-shot/paas-1962-apply-8032-patch.yml:23]: the long sobs of the violins of autumn

**Review by laurentfufu — CHANGES_REQUESTED** [review]

File: packages/one-shot/paas-1962-apply-8032-patch.yml (hunk 1 above)
Comment (kevinvandel) [reply packages/one-shot/paas-1962-apply-8032-patch.yml]: fixed

File: packages/one-shot/paas-1962-apply-8032-patch.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - cmd[proc, cp]: |-
+        provisioning_version=$(grep -R --include bundle.info provisioning /data/digital-factory-data/bundles-deployed/ | awk -F "/" '{print $NF}')
+        versions_count=$(echo $provisioning_version | wc -w)
+        if [ $versions_count -lt 1 ];then
+          echo "Provisioning bundle isn't installed"
+          exit 1
+        elif [ $versions_count -gt 1 ]; then
+          echo "Mulitple versions of provisioning bundle are installed"
+          exit 1
+        fi
```
[the remark this answers is not in the document]
Comment (kevinvandel) [reply packages/one-shot/paas-1962-apply-8032-patch.yml]: fixed

File: packages/one-shot/paas-1962-apply-8032-patch.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  - installPatch
+  - checkPatchInstallation
+
+actions:
+  checkJahiaVersion:
+    - getJahiaVersion
+    - if (${globals.jahiaVersion} != "8.0.3.0"):
+        - return:
+            type: success
+            message: "Jahia version is ${globals.jahiaVersion}, not 8.0.3.0 ; aborting"
```
Comment (laurentfufu) [inline packages/one-shot/paas-1962-apply-8032-patch.yml:23]:
```suggestion
            message: "Jahia version is ${globals.jahiaVersion}, not 8.0.3.0 ; nothing to do"
```
for the record :trollface:

**Review by laurentfufu — APPROVED** [review]

File: packages/one-shot/paas-1962-apply-8032-patch.yml (hunk 3 above)
Comment (kevinvandel) [reply packages/one-shot/paas-1962-apply-8032-patch.yml:23]: Number of fucks given : :zero:

**Pull request merged**